### PR TITLE
chore: package updates

### PR DESCRIPTION
**Pull request by rebelchris**
## Changes

Some package updates that seemed safe enough

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://chore-package-updates.preview.app.daily.dev

### CODE REVIEW

File: packages/storybook/package.json [hunk 1]
```diff
@@ -55,7 +54,7 @@
     "postcss-nesting": "^13.0.0",
     "postcss-rem-to-responsive-pixel": "^6.0.2",
     "postcss-simple-vars": "^7.0.1",
-    "serve": "^13.0.4",
+    "serve": "^14.2.4",
```
[the remark this answers is not in the document]
Comment (omBratteng) [reply packages/storybook/package.json:57]: Ah, for preview. Just didn't see it in the other files

[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/storybook/package.json:57]: Probably not ill remove it 👍

File: packages/storybook/package.json [hunk 2]
```diff
@@ -5,7 +5,6 @@
   "scripts": {
     "dev": "storybook dev -p 6006",
     "build": "storybook build",
-    "preview-storybook": "serve storybook-static",
```
Comment (nensidosari) [inline packages/storybook/package.json:8]: This removal intentional?

**Review by nensidosari — APPROVED** [review]
Looks good

**Review by omBratteng — APPROVED** [review]

File: packages/storybook/package.json (hunk 2 above)
Comment (rebelchris) [reply packages/storybook/package.json:8]: Yeah we don't particularly use it, and adds another dependency

**Pull request merged**